src/components/Tutorial/HeroMetadata.vue

```diff
@@ -13,17 +13,24 @@
     <div
       class="item"
       v-if="estimatedTimeInMinutes"
-      :aria-label="`${estimatedTimeInMinutes} minutes estimated time`"
+      :aria-label="`
+        ${$tc(
+          'tutorials.time.minutes.full', estimatedTimeInMinutes, { count: estimatedTimeInMinutes }
+        )}
+        ${$t('tutorials.estimated-time')}
+      `"
     >
       <!-- Accessibility warning: if you remove the label above,
       also remove the aria-hidden="true" values below. -->
       <div class="content" aria-hidden="true">
         <div class="duration">
           {{ estimatedTimeInMinutes }}
-          <div class="minutes">min</div>
+          <div class="minutes">{{ $tc(
```

Comment:
Is it guaranteed that the number always goes in front of "minute"? I think we should localize the whole thing together.

Reply:
This was a difficult thing to fix because there is HTML in the middle but I think I managed a very good solution :) Please test it out again